app/models/video.rb

```diff
@@ -8,4 +9,16 @@ def self.search_by_title(search_term)
     Video.where("title LIKE ?", "%#{search_term}%").order(created_at: :desc)
   end
 
+  def average_rating
```

Comment:
You could use `ActiveRecord#average` method - Read more [here](http://api.rubyonrails.org/classes/ActiveRecord/Calculations.html#method-i-average)